From `BUILDING.md`, `make moby-cli-link` causes `No rule to make target` error

Location: https://github.com/docker/compose-ecs/blob/main/BUILDING.md#:~:text=help%20with%20this%3A-,make%20moby%2Dcli%2Dlink,-This%20will%20create

```
make moby-cli-link
make: *** No rule to make target 'moby-cli-link'.  Stop.
```

Running just `make` works.

Solutions:
* Remove it from the doc.
* Add the build target to the makefile.

--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 bin/
 dist/
+/.idea/
 /.vscode/

--- a/BUILDING.md
+++ b/BUILDING.md
@@ -22,15 +22,7 @@ make
 This will output a CLI for your host machine in `./bin`.
 
 You will then need to make sure that you have the existing Docker CLI in your
-`PATH` with the name `com.docker.cli`. A make target is provided to help with
-this:
-
-```console
-make moby-cli-link
-```
-
-This will create a symbolic link from the existing Docker CLI to
-`/usr/local/bin` with the name `com.docker.cli`.
+`PATH` with the name `com.docker.cli`.
 
 You can statically cross compile the CLI for Windows, macOS, and Linux using the
 `cross` target.

